lib/Stripe/BitcoinTransaction.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class Stripe_BitcoinTransaction extends Stripe_ApiResource
```

Comment:
Yes. They're only createable through Customer action (eg. transferring funds to the receiver). I'm certainly open to having the BitcoinTransaction sublist on BitcoinReceiver just be associative arrays, though